File: despatch.WEB/app/src/swagger/models/user-urls-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { UrlRequest } from './url-request';
import { UserRequest } from './user-request';
export interface UserUrlsRequest extends BaseEntityRequest{
  url?: UrlRequest;
  urlId?: string;
  user?: UserRequest;
  userId?: string;
}

File: despatch.Domain/Models/Default/Request/UserUrlsRequest.cs
namespace despatch.Domain.Models.Default.Request
{
    public class UserUrlsRequest : BaseEntityRequest
    {
        public string UserId { get; set; }
        public string UrlId { get; set; }

        public virtual UserRequest User { get; set; }
        public virtual UrlRequest Url { get; set; }
    }
}
